pandas/tests/interchange/test_impl.py

```diff
@@ -309,11 +313,21 @@ def test_datetimetzdtype(tz, unit):
     tm.assert_frame_equal(df, from_dataframe(df.__dataframe__()))
 
 
-def test_interchange_from_non_pandas_tz_aware():
+def test_interchange_from_non_pandas_tz_aware(request):
     # GH 54239, 54287
     pa = pytest.importorskip("pyarrow", "11.0.0")
     import pyarrow.compute as pc
 
+    if is_platform_windows() and is_ci_environment():
```

Comment:
use _require_timezone_database?

Reply:
Yeah can do this in a follow up. It's current centralized in `test_arrow.py`